src/viur_cli/deploy.py

```diff
@@ -49,8 +54,8 @@ def deploy(action, name, additional_args):
                 with open(yaml_file, "r") as source_file:
                     try:
                         data = yaml.safe_load(source_file)
-                    except:
-                        raise ValueError()
+                    except ValueError as e:
```

Comment:
This try...except has no effect. It can easily be removed, so any Exception raised by safe_load is raised as well (which is currently the same behavior)